cibuild.cmd

```diff
@@ -1,5 +1,148 @@
-@echo off
+@if not defined _echo echo off
+setlocal
 
-echo cibuild.cmd invoked from the xplat branch--calling RebuildWithLocalMSBuild.cmd
+:parseArguments
+if "%1"=="" goto doneParsingArguments
+if /i "%1"=="--scope" set SCOPE=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--target" set TARGET=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--host" set HOST=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--skip-bootstrap" set BOOTSTRAP_ONLY=true&& shift && goto parseArguments
 
-RebuildWithLocalMSBuild.cmd
+:: Unknown parameters
+goto :usage
+
+:doneParsingArguments
+
+if "%SCOPE%"=="Compile" (
+    set TARGET_ARG=Build
+) else (
+    set TARGET_ARG=BuildAndTest
+)
+
+:: Assign target configuration
+
+:: Default to full-framework build
+if not defined TARGET (
+    set TARGET=Desktop
+)
+
+set BUILD_CONFIGURATION=
+if /i "%TARGET%"=="CoreCLR" (
+    set BUILD_CONFIGURATION=Debug-NetCore
+) else if /i "%TARGET%"=="Desktop" (
+    set BUILD_CONFIGURATION=Debug
+) else (
+    echo Unsupported target detected: %TARGET%. Aborting.
```

Comment:
Could and maybe should be added. I'm pretty sure our CI machines don't have Mono installed so the second phase of the build wouldn't work as is. Not sure if it's worth the effort. @radical -- thoughts?